src/chrome/content/rules/Samsung.com.xml

```diff
@@ -0,0 +1,172 @@
+<!--
+	Other Samsung rulesets:
+
+		- AllShare_Play.xml
+		- Samsung_Apps.xml
+		- Samsung_CN.xml
+		- Samsung_Direct.xml
+		- Samsung_Indiaestore.xml
+		- Samsung_Fire.xml
+		- Samsung_Knox.xml
+		- Samsung_mySingle.xml
+		- Samsung_Sem.xml
+		- Samsung_Techwin.xml
+
+	samsungbarbarian.appspot.com
+
+	Nonfunctional domains:
+
+		- samsung.com subdomains:
+
+			- help.content *
+			- m		(503, akamai)
+			- origin *
+
+		- (www.)samsungmobilepress.com *
+
+	* Times out
+
+	Problematic domains:
+	
+		- samsung.com subdomains:
+
+			- ^		(Too many redirects)
+			- chaton	(akamai)
+			- nmetrics	(mismatched, CN: *.112.2o7.net)
+			- shop.us	(some paths redirect to http)
+			- smm	(mismatched)
+			- stg-account	(MCB at https://stg-account.samsung.com/membership/service/getServieInfoList.do and break the auto recirect for https://stg-account.samsung.com/membership/signUp.do and so on.)
+
+	Partially covered samsung.com subdomains:
+		- apps	(redirects to main page of sumsung by country)
+		- opensource	(at least some pages redirect to http)
+		- shop.us	(display cart requests redirect to http)
+
+	Fully covered domains:
+
+		- samsung.com subdomains:
+
+			- (www.)	(^ → www)
+			- account
+			- allshareplay
+			- content
+			- img.content
+			- pub.content
```

Comment:
Why has img.content and pub.content has been removed from the rules?
If they have issues or do no loner exist, the comments should also be updated.